### ISSUE TITLE
No focus on URL bar in new tab

### ISSUE THREAD

**Issue by cem-cassdoc** [jimschubert/NewTab-Redirect#181]
So when I do a new tab, there no longer is focus in the url bar so I can't start typing a search or URL. The custom page is fine.

**Comment by cem-cassdoc** [comment]
chrome version: 83.0.4103.106 (Official Build) (64-bit)
OS: OSX Mojave

**Comment by johndmeyer** [comment]
I am seeing this issue on Win 10 and Chrome Version 83.0.4103.97 (Official Build) (64-bit). I really liked this feature - I hope it gets fixed soon.

**Comment by ceuk** [comment]
+1 tested on the latest version of chrome running on Arch and Windows 10 and Catalina ( I have many computers.. :D)

**Comment by lavandongen** [comment]
I have this problem also, both on Mac and Windows running the latest version of google Chrome

**Comment by Velloria** [comment]
Yes I'm also having this issue. The extension used to have the url selected and the caret in the url bar. Now it does for a split second, then immediately goes to the google search box. It's annoying because I type a new url only to find out half of it is in the url box and the other half in the search box.

**Comment by jordangeorge** [comment]
I also have this problem. I have to hit "Tab" after creating a new tab with Cmd+T  to get my cursor in the search bar (this is with the "new tab redirect" extension, blank page) when before I was using Cmd+T and was able to start typing with "new tab redirect" being at chrome://bookmarks/ which i prefer a lot more. I can't use chrome://bookmarks/ anymore because I will end up tabbing through the bookmarks. Or I could have it put have to use my mouse to click the search bar. Such a small thing but it has really messed with my productivity.

**Comment by jonnyruzek** [comment]
+1 on this issue with version 83+ on both Chrome, Canary, and Edge.

This is kind of a deal-breaker for me, so I hope a fix is in the works. 

For macOS users, a temporary workaround is to CMD+L to get address bar focus.

**Comment by ceuk** [comment]
@jonnyruzek I don't want to be the tone police and there's nothing wrong with reporting bugs but I'd remember this is free, open source software maintained for the most part by people with full time jobs in their spare time :)

**Comment by jonnyruzek** [comment]
@ceuk My apologies for whatever tone came through - I love the extension and use it extensively. I'd hate to have to switch to another browser or use something else since this one has been really, really good!

**Comment by zzpenguin** [comment]
Just updated to Chrome 84 and it's now working again

**Comment by johndmeyer** [comment]
Working for me again now too! Yay!

**Comment by jonnyruzek** [comment]
Awesome, working on Chrome 84.0.4147.89 for me too. Thanks!

**Comment by femto-code** [comment]
Confirming that it works again on MS Edge 84.0.522.40. Very helpful extension!

**Comment by jimschubert** [comment]
The reason why this occurs is addressed [in the readme](https://github.com/jimschubert/NewTab-Redirect#omnibar-support).
